docs/quickstart.md

```diff
@@ -126,7 +126,7 @@ Dredd's [hooks](hooks.md) enable you to write some glue code in your favorite la
 For more complex example applications, please refer to:
 
 - [Express.js example application](http://github.com/apiaryio/dredd-example)
-- [Ruby on Rails example application](https://github.com/theodorton/dredd-test-rails)
+- [Ruby on Rails example application](https://github.com/apiaryio/dredd-test-rails)
```

Comment:
@theodorton moved the repo to https://gitlab.com/theodorton/dredd-test-rails. @netmilk do you know which one of the repos (https://gitlab.com/theodorton/dredd-test-rails vs https://github.com/apiaryio/dredd-test-rails) is the one we should recommend in docs?